lib/src/operations/cardano_derive_address_operation.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:typed_data';
+import 'package:buffer/buffer.dart';
+import 'package:ledger_cardano/src/operations/cardano_ledger_operation.dart';
+import 'package:ledger_cardano/src/operations/complex_ledger_operations.dart';
+import 'package:ledger_cardano/src/operations/ledger_operations.dart';
+import 'package:ledger_cardano/src/utils/cardano_networks.dart';
+import 'package:ledger_cardano/src/utils/hex_utils.dart';
+
+class CardanoDeriveAddressOperation extends ComplexLedgerOperation<String> {
+  final List<int> bip32SpendingPath;
+  final List<int> bip32StakingPath;
+  final CardanoNetwork network;
+
+  CardanoDeriveAddressOperation({
+    required this.bip32SpendingPath,
+    required this.bip32StakingPath,
+    required this.network,
+  });
+
+  @override
+  Future<String> invoke(LedgerSendFct send) async {
+    final ByteDataWriter writer = ByteDataWriter();
+
+    writer.writeUint8(0);
```

Comment:
Regarding magic numbers (hard coded numbers). Please follow either of the 2 approaches:
1. Best approach: add it as a static const variable (private if it is only relevant for this file/class)
2. Ok approach: add a comment on the line or above it to say what that number represents.

P.s. I have no idea what this `0` here means and why we're adding it.

Please add a `Magic Numbers` section in the guideline with the 2 accepted approaches please.

Reply:
done